_mental_models/saying-no.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: saying-no
+name: Saying "No"
+benefit: Better Decisions
+summary: Saying "yes" to requests because you don't want to upset people can cause problems.
+prerequisites: Confidence, certainty
```

Comment:
I'm not sure the training needs confidence or certainty. The training will provide those.